projects/core/directives/hint/hint-hover.directive.ts

```diff
@@ -76,6 +76,9 @@ export class TuiHintHover extends TuiDriver {
     @Input()
     public tuiHintHideDelay: TuiHintOptions['hideDelay'] = this.options.hideDelay;
 
+    @Output('tuiHintHoverChange')
+    public readonly tuiHintHoverChange = new EventEmitter<boolean>();
```

Comment:
I think this should be called `tuiHintVisible` and it should go into main hint directive, somewhere here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/core/directives/hint/hint.directive.ts#L77